source/transactions-convenient-api/transactions-convenient-api.md

```diff
@@ -128,11 +129,23 @@ This method should perform the following sequence of actions:
 6. If the callback reported an error:
     1. If the ClientSession is in the "starting transaction" or "transaction in progress" state, invoke
         [abortTransaction](../transactions/transactions.md#aborttransaction) on the session.
+
     2. If the callback's error includes a "TransientTransactionError" label and the elapsed time of `withTransaction` is
-        less than 120 seconds, jump back to step two.
+        less than 120 seconds, calculate the backoffMS to be
+        `jitter * min(BACKOFF_INITIAL * (1.25**retry), BACKOFF_MAX)` where:
```

Comment:
Maybe this was a miss when CSOT was implemented, but should this also be updated to mention timeoutMS?